src/studios/airwaze/index.rst

```diff
@@ -95,8 +91,18 @@ This application uses Spring Boot and Spring Data.  When the project boots up, *
 The ``import.sql`` file in this project contains SQL statements that import CSV data into SQL tables. Please take a minute to review ``import.sql`` and the two ``.csv`` files.
 
 - Review the ``import.sql`` file in the airwaze project
-- Be sure that the ``import.sql`` points to **your** local copies of the ``airports.csv`` and ``routes.csv``. It needs to be the "full path" to each file on your local computer.
-- You will need to change the path value for both files listed. ``/Users/YOUR-USERNAME/YOUR-PATH/airwaze-studio/routes.csv``
+
+We are using a PostGIS docker container. Althought it is installed on our computer, docker is a virtualization/containerization tool, so our PostGIS container is unaware of any other files on our computer. We will need to COPY our CSV files to a location our PostGIS container can work with.
+
+Luckily docker gives us an easy way to do this.
+
+- From the root directory of airwaze-studio run ``$ docker cp Airports.csv postgis:/tmp``
+- And also run ``$ docker cp routes.csv postgis:/tmp``
+
+The ``cp`` command in docker stands for copy. We are copying the two CSV files into a directory called /tmp inside of our container.
+
+- Be sure that the ``import.sql`` points to the copies of ``airports.csv`` and ``routes.csv`` in your container.
+- You shouldn't need to change anything in the import.sql file. ``/tmp/routes.csv``
```

Comment:
This wording seems weird. Should `/tmp/routes.csv` be removed the end of the line?

Reply:
I removed the whole line. We don't need to tell them that nothing needs to be changed to that file, especially since we ask them to review the file to figure out what is going on.